indexing-service/src/main/java/org/apache/druid/indexing/overlord/ImmutableWorkerInfo.java

```diff
@@ -57,6 +63,7 @@ public ImmutableWorkerInfo(
       @JsonProperty("worker") Worker worker,
       @JsonProperty("currCapacityUsed") int currCapacityUsed,
       @JsonProperty("currParallelIndexCapacityUsed") int currParallelIndexCapacityUsed,
+      @JsonProperty("currTypeSpecificCapacityUsed") @Nullable Map<String, Integer> typeSpecificCapacityMap,
```

Comment:
Please use the same name for the argument, the JSON serialized field and the class member for readability.

Also, this name is a little ambiguous, use something else like `currCapacityUsedByTaskType`.

Reply:
Updated